api/tests/Feature/AvailabilityService/AvailabilityServiceTest.php

```diff
@@ -400,7 +400,8 @@ private function returnEventDataByEventTypeId($id)
             'invitee_name' => 'Invitee Name',
             'invitee_email' => 'invitee@gmail.com',
             'timezone' => 'Europe/Kiev',
-            'start_date' => '2020-08-07 15:00:00'
+            'start_date' => '2020-08-07 15:00:00',
+            'custom_field_values' => []
```

Comment:
We need better tests to be sure the application is working.

Reply:
What do you mean?

Reply:
I mean we should test custom fields too

Reply:
@aLLeXUs , will be written